src/components/CreateList/index.js

```diff
@@ -0,0 +1,22 @@
+import { getToken } from '@the-collab-lab/shopping-list-utils';
+import { useLocalStorage } from '../../hooks/useLocalStorage';
+import { useHistory } from 'react-router-dom';
```

Comment:
@mailliwmogollon it's a good practice to have third party libraries on top of our code. So I would suggest you to move this `useHistory` import to the top and move the `useLocalStorage` import down

Reply:
Just commited this! thank you so much @Danieljara23. I find this highly important as I would import libraries as they come, which can led to an unorganized importing :+1: :100: